modules/developer-cli-odo-installing-odo-on-windows.adoc

```diff
@@ -6,29 +6,23 @@
 
 = Installing {odo-title} on Windows
 
-== Binary installation
-
-. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo.exe`] file.
-. Add the location of your `odo.exe` to your `GOPATH/bin` directory.
-
-[discrete]
-== Setting the `PATH` variable for Windows 7/8
-
-The following example demonstrates how to set up a path variable. Your binaries can be located in any location, but this example uses `C:\go-bin` as the location.
-
-. Create a folder at `C:\go-bin`.
-. Right click *Start* and click *Control Panel*.
-. Select *System and Security* and then click *System*.
-. From the menu on the left, select the *Advanced systems settings* and click *Environment Variables* at the bottom.
-. Select *Path* from the *Variable* section and click *Edit*.
-. Click *New* and type `C:\go-bin` into the field or click *Browse* and select the directory, and click *OK*.
-
-[discrete]
-== Setting the `PATH` variable for Windows 10
-
-Edit `Environment Variables` using search:
-
-. Click *Search* and type `env` or `environment`.
-. Select *Edit environment variables for your account*.
-. Select *Path* from the *Variable* section and click *Edit*.
-. Click *New* and type `C:\go-bin` into the field or click *Browse* and select the directory, and click *OK*.
+The `{odo-title}` CLI for windows is available to download as a binary and as an archive.
+
+.Procedure
+
+. Navigate to link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[] and download the `odo-windows-amd64.exe` binary or the `odo-windows-amd64.exe.zip` archive.
+. If you downloaded the archive, unzip it with a ZIP program.
```

Comment:
In multiple procedures, you have this conditional phrase in step 2 `If you <did a specific action>, <do another specific action as a result>.` This formula is tripping me up because as a user, I'm thinking, "do I need to follow the other steps if I didn't download the archive? If I didn't download the archive, am I done after step one?" 

Like in my earlier comments, I think this whole procedure wants users to download the archive file, so I would simply tell them to do so in step one.

Reply:
I have added separate sub-steps for binary vs tarball.

As above, I agree that we should only document archives in future, but don't feel I should make that decision as I may not be fully aware of all the pros/cons.